pages/pipelines/group_step.md.erb

```diff
@@ -0,0 +1,208 @@
+# Group Step
+ 
+A *group* step can contain various sub-steps, and display them in a single logical group on the Build page. For example, you can group all of your linting steps or all of your UI test steps to keep the Build page less messy. Sub-groups and nested groups are not supported.
+
+The group step also helps manage dependencies between a collection of steps, for example, "step X" [`depends-on`](/docs/pipelines/group-step#group-step-attributes) everything in "group Y".
+
+{:toc}
+ 
+A group step can be defined in your pipeline settings or your [pipeline.yml](/docs/pipelines/defining-steps) file.  
+
+Here is an example of using the group step:  
+
+```yml
+- group: "\:broom\: Linting"
+  key: "linting"
+  depends_on: docker
+  steps:
+    - name: "\:rubocop\: RuboCop"
+      command: ".buildkite/steps/rubocop"
+    - name: "\:eslint\: ESlint"
+      command: ".buildkite/steps/eslint"
+    - name: "\:flowtype\: Flow"
+      command: ".buildkite/steps/flow"
+```
+{: codeblock-file="group-step-in-pipeline.yml"} 
+
+This is how it's displayed in the UI: 
+
+<%= image "job-groups-in-UI.png", width: 1760, height: 436, alt: "Job groups displayed in the Buildkite UI" %>
+
+Only the first 100 jobs within a build header can ever be displayed in the UI, so you might not see all of your groups at all times. However, the jobs are fine and will still show up on the build page. 
+
+The group step type also groups jobs up by a `group_identifier` ([`key` attribute](/docs/pipelines/group-step#group-step-attributes)) if it exists.
```

Comment:
I still don't know what you mean by this

Reply:
Basically, you can either group things using 

```YAML
group: "linting"
```

or 

```
- group:
  label: "linting"
```

right? 

So if you ommit the group key, and just use `group: linting` syntax, you can add them in multiple places and they will be joined into one group. 

I **think** that is it?

Reply:
But is the label the identifier or the key?

Reply:
```suggestion
You can [merge groups](#group-merging) by giving them the same `label` or name.
```